src/main/scala/com/datastax/sparkstress/WriteTask.scala

```diff
@@ -274,7 +276,33 @@ class WriteWideRowByPartition(config: Config, ss: SparkSession) extends
   }
 
   def getDataFrame: org.apache.spark.sql.DataFrame = {
-    RowGenerator.getWideRowByPartitionDataFrame(ss, config.seed, config.numPartitions, config.totalOps, config.numTotalKeys)
+    RowGenerator.getWideRowByPartitionDataFrame(ss, config.seed, config.numPartitions, config.totalOps, config.numTotalKeys
+    )
   }
+}
+
+@WriteTest
+class CopyTable(config: Config, ss: SparkSession) extends
+  WriteTask[PerfRowClass](config, ss)(implicitly[RowWriterFactory[PerfRowClass]]) {
 
+  override def getTableCql(tbName: String): Seq[String] =
+    Seq(
+      s"""CREATE TABLE IF NOT EXISTS ${tbName}_copy
+       |(store text,
+       |order_time timestamp,
+       |order_number uuid,
+       |color text,
+       |size text,
+       |qty int,
+       |PRIMARY KEY (store, order_time, order_number))
+     """.stripMargin)
+
+  override def destination: TableLocation = TableLocation(config.keyspace, s"${config.table}_copy")
```

Comment:
I guess since the parent definition of `destination` will never be modified away from `config.table` it's safe to re-reference `config.table` here, otherwise I would want to reference `destination.table`.

Reply:
Not sure I follow? We override and do not call the parent?

Reply:
yeah, after thinking about it my original comment doesn't make sense here because of the way we're defining destination with `def`. nvm